From stable-bounces@linux.kernel.org  Thu Jul  6 13:00:36 2006
Date: Thu, 06 Jul 2006 14:26:44 -0400
From: Michael Krufky <mkrufky@linuxtv.org>
To: stable@kernel.org
Cc: v4l-dvb maintainer list <v4l-dvb-maintainer@linuxtv.org>, Andrew de Quincey <adq_dvb@lidskialf.net>
Subject: v4l/dvb: stradis: dont export MODULE_DEVICE_TABLE

From: Andrew de Quincey <adq_dvb@lidskialf.net>

This patch prevents the stradis driver from breaking all
other saa7146 devices by removing the autodetection based
on PCI subsystem ID 0000:0000 (no eeprom).  Users that
want to use the stradis driver will have to manually
insert the module, or specify it in modprobe.conf

Signed-off-by: Andrew de Quincey <adq_dvb@lidskialf.net>
Signed-off-by: Michael Krufky <mkrufky@linuxtv.org>
Signed-off-by: Chris Wright <chrisw@sous-sol.org>
---

 drivers/media/video/stradis.c |    1 -
 1 file changed, 1 deletion(-)

--- linux-2.6.17.3.orig/drivers/media/video/stradis.c
+++ linux-2.6.17.3/drivers/media/video/stradis.c
@@ -2180,7 +2180,6 @@ static struct pci_device_id stradis_pci_
 	{ 0 }
 };
 
-MODULE_DEVICE_TABLE(pci, stradis_pci_tbl);
 
 static struct pci_driver stradis_driver = {
 	.name = "stradis",
